src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx

```diff
@@ -24,14 +24,14 @@ interface ViewerWrapperProps {
     axisClipSliders: boolean;
   };
   onView3DCreated: (view3d: View3d) => void;
-  setAxisClip: (axis: string, minval: number, maxval: number, isOrthoAxis: boolean) => void;
+  setAxisClip: (axis: "x" | "y" | "z", minval: number, maxval: number, isOrthoAxis: boolean) => void;
```

Comment:
use AxisName here?

Reply:
somewhere in `shared/`, like maybe `shared/enums` might be a place for such a definition, but it's not critical for this PR

Reply:
of course, my other thought is to make it a type that comes out of volume-viewer but there's probably good reason to keep the two layers separated (the ui type might need to diverge from the lower level type, and you'd want a uiAxisToVolviewerAxis() function before passing the axis name down to the lower level interface).  I haven't looked hard enough at the code to see if that would be needed...

Reply:
I've honestly been waiting for a critical mass of types without an obvious home to justify creating shared/types.ts, but there have been pretty few uncategorized utility types so far.